articles/react-native-firebase-facebook-authentication/index.md

```diff
@@ -0,0 +1,460 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/react-native-firebase-facebook-authentication/
+title: Facebook OAuth using Firebase in React Native
+description: This tutorial will give readers a detailed guide on how to implement Facebook OAuth using Firebase in a Non-Expo React Native Application.
+author: mohan-raj
+date: 2021-03-31T00:00:00-10:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/react-native-firebase-facebook-authentication/hero.jpg
+    alt: Facebook OAuth using Firebase in React Native Image
+---
+In this tutorial, we will learn how to authenticate users with their Facebook accounts using the authentication module in Firebase in a Non-Expo React Native application.
+<!--more-->
+
+To learn more about Firebase, refer to [this link](https://en.wikipedia.org/wiki/Firebase).
+
+### Prerequisites
+
+We'll not cover the basics of React and React Native in this tutorial. If you are not comfortable with the basics, we highly recommended going over this [tutorial](https://reactnative.dev/docs/tutorial) before you continue further.
+
+### Overview
+
+We'll be going through these steps in this article:
+
+1. [Development environment](#development-environment).
+2. [Cloning the starter code](#cloning-the-starter-code).
+3. [Setting up the Firebase project](#setting-up-the-firebase-project).
+4. [Setting up Facebook App](#setting-up-facebook-app)
+5. [Setting up Firebase Authentication](#setting-up-firebase-authentication).
+6. [Sign-in](#sign-in)
+7. [Display authenticated screen](#display-authenticated-screen)
+8. [Sign out](#sign-out)
+9. [Recap](#lets-recap)
+
+### Development environment
+
+> **IMPORTANT** - We will not be using [Expo](https://expo.io/) in our project.
+
+Follow [this documentation](https://reactnative.dev/docs/environment-setup) to set up the development environment.
+
+Make sure you're following the React Native CLI Quickstart, not the Expo CLI Quickstart.
+
+![Env Setup](/engineering-education/react-native-firebase-facebook-authentication/env_setup.png)
+
+### Cloning the starter code
+
+Clone the starter code from this [GitHub repository](https://github.com/zolomohan/rn-firebase-phone-auth-starter).
+
+You can check out the final code from this [GitHub Repository](https://github.com/zolomohan/rn-firebase-phone-auth).
```

Comment:
Please change the GitHub repo to the correct repository. This one points to the Phone Auth code.

Reply:
😅😅 my bad. Fixed it. 
Good catch.